packages/manager/apps/hub/package.json

```diff
@@ -22,7 +22,7 @@
     "@ovh-ux/manager-account-sidebar": "^2.0.0 || ^3.0.0",
     "@ovh-ux/manager-at-internet-configuration": "^0.0.0 || ^1.0.0",
     "@ovh-ux/manager-banner": "^1.1.3",
-    "@ovh-ux/manager-billing": "^0.0.0 || ^1.0.0",
+    "@ovh-ux/manager-billing": "^1.0.0 || ^2.0.0",
```

Comment:
Hub will be missing peerDeps required by `@ovh-ux/manager-billing` but I'm not sure we should add all those as they will not be used

Reply:
Why hub will be missing peerDeps? by default, importing manager-billing won't import all modules. It only imports legacy code. To import billing code, need to specify it explicitly like import { billingManagement } from '@ovh-ux/manager-billing';. Hub will not be importing all billing module code.

Reply:
My bad what I meant is that Hub will be missing dependencies. Even though we don't import billing code, some peer deps are still required by `@ovh-ux/manager-billing` which would need to be declared as hub dependency as it uses the module. I wonder if in that case we should instead use the `optionalDependencies`

Reply:
Ok its very clear now. You are right. Hub will not have all the dependencies mentioned but they are not required. Still, it will be very confusing. I think its better to separate legacy billing components from the billing module. Thanks a lot.

Reply:
@radireddy Is it planned in a separate PR ? Because currently some dependencies are missing